plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/ReusableConnectionFactory.java

```diff
@@ -128,7 +128,7 @@ public void close()
         delegate.close();
     }
 
-    private final class CachedConnection
+    final class CachedConnection
```

Comment:
Undo

Reply:
needed to be able to access this class in a test (package-private scoped)